src/applications/ivc-champva/10-7959C/config/submitTransformer.js

```diff
@@ -27,14 +27,6 @@ export default function transformForSubmit(formConfig, form) {
       copyOfData.applicantName?.middle?.charAt(0) ?? '';
   }
 
-  // go from medigapPlanA -> A
-  if (copyOfData.primaryMedigapPlan) {
-    copyOfData.primaryMedigapPlan = copyOfData.primaryMedigapPlan.slice(-1);
-  }
-  if (copyOfData.secondaryMedigapPlan) {
-    copyOfData.secondaryMedigapPlan = copyOfData.secondaryMedigapPlan.slice(-1);
-  }
-
```

Comment:
Renamed all medigap keys to remove the prefix from the start, this logic is now unnecessary.